apps/test/unit/code-studio/pd/application/TeacherApplicationTest.js

```diff
@@ -83,6 +84,7 @@ describe('TeacherApplication', () => {
     const teacherApplication = isolateComponent(
       <TeacherApplication {...defaultProps} allowPartialSaving />
     );
+    const storage = new FakeStorage();
```

Comment:
I would suggest moving this into defaultProps, to make sure you are always using the fake storage in unit tests.

Reply:
Another thing that's awkward about this: it doesn't work on `mount` because the storage doesn't persist into FormController ... I need to move the `mount` tests anyways into integration tests